src/prefect/cli/auth.py

```diff
@@ -477,17 +478,32 @@ def status():
         except Exception as exc:
             click.echo(f"Your authentication is not working: {exc}")
 
-    if client._api_token:
+    if AUTH_TOKEN_SETTINGS_PATH.exists():
         click.secho(
-            "You are logged in with an API token. These have been deprecated in favor "
-            "of API keys."
-            + (
-                " Since you have set an API key as well, this will be ignored."
-                if client.api_key
-                else ""
-            ),
+            "The authentication tokens settings path still exists. These have been "
+            "removed in favor of API keys. We recommend purging old tokens with "
+            "`prefect auth purge-tokens`",
             fg="yellow",
         )
 
-    if not client._api_token and not client.api_key:
-        click.secho("You are not logged in!", fg="yellow")
+    if config.cloud.get("auth_token"):
+        if os.environ.get("PREFECT__CLOUD__AUTH_TOKEN"):
```

Comment:
hmm will this always warn when we pass `PREFECT__CLOUD__AUTH_TOKEN` to flow run containers?

https://github.com/PrefectHQ/prefect/pull/5140/files#diff-3803ec5e942824e9426b9fd46beb5f7cbd4b1d626fbe1bd803f2bdc06f68df2fR580

Reply:
ahhh nvm okay i guess this is just the auth cli